stdlib/asm/math/fri.masm

```diff
@@ -0,0 +1,233 @@
+use.std::math::ext2
+
+# Given a stack in the following initial configuration [a1,a0,b1,b0,c1,c0,d1,d0,...] the following
+# procedure computes (a + b + ((a - b) * c * d^(-1))) with the assumption that d1 is equal to 0
```

Comment:
Is this description still accurate? Seems like we return 2 values at the end and also there seems to be a division by 2 happening.

Reply:
Yes, the expression is in the extension field so it will return 2 elements. I will fix it to make it clearer.
Thank you, there is a missing division by 2 indeed.